english/make-hot-chocolate/hot-choc.md

```diff
@@ -0,0 +1,21 @@
+You have a word with the marshmallow who tried to grab the lighter.
+
+You both chat away happily, he apologises for going a bit crazy.
+
+He didn't realise it was you, he likes you!
+
+So much so he offers a bit of mallow from his walls to add to
+the cup of cocoa you find in your hand.
+
+You think that's a bit gross as he's essentially offering some
+of himself to you but you go with it.
+
+That drink turns out to be the best hot chocolate and marshmallows ever!
+
+Once you finish it you think what shall I do next?
+
+[Go for a run to burn off those sugary calories](create-your-own-adventure/english/run/run.md).
+
+[Do a spot of yoga](create-your-own-adventure/english/yoga/yoga.md).
+
+[Look for a way to leave](https://github.com/luddles82/create-your-own-adventure/blob/master/english/find-exit/leave.md)
```

Comment:
Please don't include an absolute to your own GitHub profile. Instead, it should be a relative link like your other ones.